Formula/blush.rb

```diff
@@ -0,0 +1,24 @@
+class Blush < Formula
+  desc "Grep with colours"
+  homepage "https://github.com/arsham/blush"
+  url "https://github.com/arsham/blush/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "6db6b033bb5d4c4ac350b36b82d79447d5b91509db3a5eceb72ecb9484495e54"
+  license "MIT"
+  head "https://github.com/arsham/blush.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=#{version}")
+  end
+
+  test do
+    # Display issue on CI
+    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Why does it need a display if it's just grep?

Reply:
Should probably fix the failure instead

Reply:
yeah, I might need some help on this. cc @cho-m @danielnachun 🙏